ariadne/executable_schema.py

```diff
@@ -48,16 +46,6 @@ def join_type_defs(type_defs: List[str]) -> str:
     return "\n\n".join(t.strip() for t in type_defs)
 
 
-def build_and_extend_schema(ast: DocumentNode) -> GraphQLSchema:
-    schema = build_ast_schema(ast)
-    extension_ast = extract_extensions(ast)
-
-    if extension_ast.definitions:
-        schema = extend_schema(schema, extension_ast)
-
-    return schema
-
-
 EXTENSION_KINDS = [
```

Comment:
@nikordaris I think this list of `EXTENSION_KINDS` can also be removed, from what I can tell it's only used in `extract_extensions`, which is no longer present.

Hope this gets merged and released soon! 😄

Reply:
Nice catch, I'll take a look tomorrow morning and get it removed.